draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -358,7 +358,18 @@ explore how the capacity is shared among the competing flows.
 
 ### Short Flows
 
-(TODO: Discuss short and long flows)
+A great deal of congestion control analysis concerns the steady-state behavior
+of long flows. However, many internet flows are relatively short-lived. If they
+never experience a packet loss, they remain in the "slow start" mode of
+operation {{?RFC5681}} that features exponential congestion window growth.
+
+For decades, most proposals kept slow-start unchanged and innovated only in the
+post-loss phase of congestion control. However, more recent work (like Hystart++
+{{?RFC9406}}) has sought to mitigate the poor latency effects of unfettered
+exponential window growth.
+
```

Comment:
That's one way to describe Hystart. That's not necessarily what I would have written.

Fundamentally, Hystart is about trying to detect the bottleneck CWIN without incurring packet losses, by stopping exponential growth when a delay increase is detected. This is a great idea, but it turns out that it can "stop too soon" if delay increases happen that are not due to congestion. Hystart++ tries to mitigate that by not going immediately into "congestion avoidance" after the first delay increase, but instead going to a "more cautious exponential growth" . So Hystart++ is actually a bit more aggressive than the original Hystart...

Not that I have a better way to say that...

Reply:
There's no Hystart RFC, so I thought it would be easier to skip over all that history and just say "here is something a bit different from slow start, for once"

Reply:
Yes, but then BBR also has a different slow start test. It measures the bottleneck bandwidth during slow start, and stop increasing the pacing rate if the measured throughput did not increase significantly during the last 4 RTT. That, too, has the goal of "stopping exponential growth ". So maybe mention both...

Reply:
This might not be the correct vector for this section. SHORT flows m- by which I mean ones with little data, do not need Hystart, Careful Resume, etc. They are short, they start, they send, they end. The next piece is where I think this ought to lead...